rust/services/call/server_lib/src/preflight.rs

```diff
@@ -13,9 +13,9 @@ use crate::{
 pub enum Error {
     #[error(transparent)]
     Preflight(#[from] PreflightError),
-    #[error("Gas meter error: {0}")]
+    #[error("Refunding gas: {0}")]
     GasMeter(#[from] GasMeterError),
```

Comment:
```suggestion
    RefundingGas(#[from] GasMeterError),
```

to be consistent with your change in `proving.rs`?